configurations-conception-communes/en-tete-global/menu-site.md

```diff
@@ -249,6 +250,10 @@ title: "Menu thématique"
 
 <h2 id="changements">Derniers changements</h2>
 <dl class="dl-horizontal">
+<dt>
+    <time datetime="2025-07-08" class="link-muted">2025-07-08</time>
```

Comment:
```suggestion
    <time>2025-07-08</time>
```
The date is already in ISO format, there's no need to be more explicit for the browser. The class doesn't do much since it's looking for a `<a>` tag.